src/Websites/Websites/Cmdlets/DeploymentSlots/NewAzureWebAppSlot.cs

```diff
@@ -73,7 +73,7 @@ public class NewAzureWebAppSlotCmdlet : WebAppBaseClientCmdLet
         [ValidateNotNullOrEmpty]
         public string AseName { get; set; }
 
-        [Parameter(Position = 9, Mandatory = false, HelpMessage = "Resource group of Application Service environment")]
+        [Parameter(Position = 10, Mandatory = false, HelpMessage = "Resource group of Application Service environment")]
```

Comment:
Updating the position of a paramter will cause breaking changes. We need to follow the breaking change process.